daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/examples/NestedUnion/generated_code.c

```diff
@@ -365,7 +368,7 @@ data_parseSelf(data *instance, PState *pstate)
         break;
     default:
         // Should never happen because initChoice would return an error first
-        error.d64 = (int64_t)instance->_choice;
+        error.arg.d64 = (int64_t)instance->_choice;
```

Comment:
Hmmm. I'm not seeing a change to a struct such that there is now an "arg" field you have to navigate. Did I just miss it?

Reply:
I changed that struct in a C portability fix commit that went in last May...

<https://github.com/apache/daffodil/commit/8c95b39f5e365ae15657bcdc4686b44689b1fbb9#diff-6df97fc1bdf91db52173653361b5a40e5e075b80312a69912aa23e141dad57c6>

However, these "example" generated code files aren't updated automatically, so I didn't re-generate this example file until now.